inst/include/Rcpp/DottedPair.h

```diff
@@ -35,14 +35,11 @@ RCPP_API_CLASS(DottedPair_Impl),
 	DottedPair_Impl(SEXP x) {
 	    Storage::set__(x) ;
 	}
-  #if defined(HAS_VARIADIC_TEMPLATES)
-		template <typename... T>
-		DottedPair_Impl(const T&... args) {
-			Storage::set__(pairlist(args...));
-		}
-	#else
-		#include <Rcpp/generated/DottedPair__ctors.h>
-	#endif
+    template <typename... T>
+	DottedPair_Impl(const T&... args) {
+        Storage::set__(pairlist(args...));
+    }
+    // RM #include <Rcpp/generated/DottedPair__ctors.h>
```

Comment:
Leftover or reminder?

Reply:
Leftover.  Thanks for catching this.  Gone in ddefda2dc0c8aaced0ac1a4c541e24a4f1ccc94e -- and I promptly mistyped '[ci skip]' dropping the p.  Oh well.